numcodecs/tests/test_lzma.py

```diff
@@ -45,4 +54,7 @@ def test_config():
         check_config(codec)
 
     def test_repr():
-        check_repr('LZMA(format=1, check=0, preset=1, filters=None)')
+        if _py2:
+            check_repr('LZMA(format=1, check=0L, preset=1, filters=None)')
```

Comment:
Yep, you're right. Was running into this issue before updating the registration of LZMA. Now seems ok. So have reverted this change.